config/database.yml

```diff
@@ -6,13 +6,15 @@ development: &default
   encoding: unicode
   database: app_prototype_development
   username: postgres
-  pool: 5
+  pool: <%= ENV["DB_POOL"] || ENV['MAX_THREADS'] || 5 %>
```

Comment:
Two comments:
* Use single quotes in both cases please
* Use .presence (http://apidock.com/rails/v4.2.1/Object/presence) to guard against environment vars that are set to an empty string, e.g. `ENV['DB_POOL'].presence || ENV['MAX_THREADS'].presence || 5`.

```
irb(main):005:0> nil || '' || ' ' || 'A'
=> ""
irb(main):006:0> nil.presence || ''.presence || ' '.presence || 'A'.presence
=> "A"
```